CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Added
```

Comment:
This could be a single Change instead. But then again not sure anyone reads this changelog so 🤷

Reply:
Yeah but I wanted to have both the removed and added sections to show clearly the major version bump